### PR TITLE
ar: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.nl/common/ar.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Maken, aanpassen en uitpakken van Unix archieven. Typisch gebruikt voor statische bibliotheken (`.a`) en Debian pakketten (`.deb`).
+> Bekijk ook: `tar`.
+> Meer informatie: <https://manned.org/ar>.
+
+- Pak alles uit van een archief:
+
+`ar x {{pad/naar/bestand.a}}`
+
+- [t]oon inhoud van een specifiek archief:
```
Comment (leonvsc) [inline pages.nl/common/ar.md:11]:
```suggestion
- [T]oon inhoud van een specifiek archief:
```

Not sure if this needs to be a capital letter?

File: pages.nl/common/ar.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`ar t {{pad/naar/bestand.ar}}`
+
+- Ver[r]vang of voeg specifieke bestanden toe aan een archief:
```
Comment (leonvsc) [inline pages.nl/common/ar.md:15]:
```suggestion
- Ve[r]vang of voeg specifieke bestanden toe aan een archief:
```

**Review by leonvsc — APPROVED** [review]
LGTM! Left 2 suggestions, not sure if they are needed.

File: pages.nl/common/ar.md (hunk 1 above)
Comment (kbdharun) [reply pages.nl/common/ar.md:11]: Since it's a mnemonic we can skip capitalising it here.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**